state/migration_import.go

```diff
@@ -68,8 +71,43 @@ func (st *State) Import(model description.Model) (_ *Model, _ *State, err error)
 		// filesystems or storage instances.
 		StorageProviderRegistry: storage.StaticProviderRegistry{},
 	}
-	if creds := model.CloudCredential(); creds != "" {
-		args.CloudCredential = names.NewCloudCredentialTag(creds)
+	if creds := model.CloudCredential(); creds != nil {
+		// Need to add credential or make sure an existing credential
+		// matches.
+		// TODO: there really should be a way to create a cloud credential
```

Comment:
This comment confused me - I guess you mean to create the cloud credential ID? Or to create a tag from (cloud, owner, name).